hooks/pre/10-reset_data.rb

```diff
@@ -57,7 +57,11 @@ def reset_candlepin
 end
 
 def pg_command_base(config, command, args)
-  "PGPASSWORD='#{config[:password]}' #{command} -U #{config[:username]} -h #{config[:host]} -p #{config[:port]} #{args}"
+  "#{command} -U #{config[:username]} -h #{config[:host]} -p #{config[:port]} #{args}"
+end
+
+def pg_env(config)
+  { 'PGPASSWORD' => config[:password] }
```

Comment:
Should we pass all `PG*` params to make the command really simple?

Reply:
yes, I was just doing this :D